<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>251</width>
      <height>271</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_1">
     <widget class="QSlider" name="horizontalSlider_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>160</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QSlider" name="horizontalSlider">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>220</y>
        <width>160</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>190</y>
        <width>160</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5"/>
    <widget class="QWidget" name="page_3">
     <widget class="QSlider" name="horizontalSliderHexagon_1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>160</width>
        <height>16</height>
       </rect>
      </property>
      <property name="maximum">
       <number>500</number>
      </property>
      <property name="value">
       <number>250</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSliderHexagon_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>160</width>
        <height>16</height>
       </rect>
      </property>
      <property name="maximum">
       <number>500</number>
      </property>
      <property name="value">
       <number>250</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>10</y>
        <width>66</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>30</y>
        <width>66</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Y</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QSlider" name="horizontalSlider_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>230</y>
        <width>160</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>0</y>
      <width>531</width>
      <height>551</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>221</width>
      <height>25</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentText">
     <string/>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Area</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Perimetr</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Centr mass</string>
    </property>
   </widget>
   <widget class="QLabel" name="Area">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>460</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="Perimeter">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>490</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>520</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
